sumo_deployments/helm/sumo-opentelemetry-demo/Chart.yaml

```diff
@@ -2,12 +2,12 @@ apiVersion: v2
 name: sumo-opentelemetry-demo
 description: A Helm chart for Kubernetes
 type: application
-version: 0.1.0
-appVersion: 0.1.0
+version: 0.1.1
+appVersion: 0.1.1
```

Comment:
Can this be done in a separate change? Do we even have any automation for publishing this Chart?

Reply:
No, chart is not published. We just have it here locally in the sources.